lib/msf/core/module.rb

```diff
@@ -207,6 +217,18 @@ def file_path
       self.class.file_path
     end
 
+    def adapted_refname
+      self.class.adapted_refname
+    end
+
+    def adapter_refname
+      self.class.adapted_refname
+    end
```

Comment:
```suggestion
    def adapted_refname
      self.class.adapted_refname
    end

    def adapter_refname
      self.class.adapter_refname
    end
```
I...don't actually understand how this was working